src/sentry/eventstore/snuba/backend.py

```diff
@@ -349,12 +352,9 @@ def __get_event_id_from_filter(self, filter=None, orderby=None, dataset=snuba.Da
 
     def __make_event(self, snuba_data):
         event_id = snuba_data[Columns.EVENT_ID.value.event_name]
-        group_id = snuba_data[Columns.GROUP_ID.value.event_name]
```

Comment:
This isn't required anymore - the `group`, `group_id` and `groups` properties will al look at the underlying `snuba_data` if `group_id` isn't explicitly passed. So since we're selecting these columns with the results we don't need to be explicit here, and `Event` can work it out